.github/workflows/test_development_environment.yml

```diff
@@ -60,3 +62,4 @@ jobs:
           APP_BUILD_VERSION: ${{ github.run_number }}
           APP_BINARY_VERSION: "dev"
           DOMAIN_FORMAT: ${{ secrets.TEST_DOMAIN_FORMAT }}
+          PROJECT_ID: ${{ secrets.EXPO_TEST_PROJECT_ID }}
```

Comment:
how do we access the project id?

Reply:
https://expo.dev/accounts/supplyallytest/projects/rationally it can be seen when u log in~